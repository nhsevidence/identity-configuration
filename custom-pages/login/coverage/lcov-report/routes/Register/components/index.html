<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for routes/Register/components</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\..\index.html">All files</a> routes/Register/components
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">65.38% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>17/26</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">66.67% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>12/18</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">54.55% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>6/11</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">70.83% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>17/24</span>
      </div>
    </div>
  </div>
  <div class='status-line medium'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="RegisterForm.js"><a href="RegisterForm.js.html">RegisterForm.js</a></td>
	<td data-value="65.38" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 65%;"></div><div class="cover-empty" style="width:35%;"></div></div></td>
	<td data-value="65.38" class="pct medium">65.38%</td>
	<td data-value="26" class="abs medium">17/26</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="18" class="abs medium">12/18</td>
	<td data-value="54.55" class="pct medium">54.55%</td>
	<td data-value="11" class="abs medium">6/11</td>
	<td data-value="70.83" class="pct medium">70.83%</td>
	<td data-value="24" class="abs medium">17/24</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Wed Mar 06 2019 12:38:07 GMT+0000 (Greenwich Mean Time)
</div>
</div>
<script src="..\..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\..\sorter.js"></script>
</body>
</html>
