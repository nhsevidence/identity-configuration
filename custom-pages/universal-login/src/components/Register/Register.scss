.mainpanel {
  display: flex;
  flex-direction: column;
  margin: auto;
  min-width: var(--col-width-mobile);
}

.screen-reader-only {
  position: absolute;
  width: 1px;
  clip: rect(0 0 0 0);
  overflow: hidden;
  white-space: nowrap;
}
