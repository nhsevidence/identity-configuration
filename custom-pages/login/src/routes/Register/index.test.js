import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'

import Register from './'

Enzyme.configure({ adapter: new Adapter() })

describe('Register components', () => {
  jest.mock('../../services/AuthApi', () => jest.fn(() => {}))

  it('should render <Register /> correctly', () => {
    expect(shallow(<Register.component />)).toMatchSnapshot()
  })
})
