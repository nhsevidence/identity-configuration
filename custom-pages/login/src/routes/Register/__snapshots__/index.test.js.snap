// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register components should render <Register /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Register />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Fieldset
        legend="Personal Information"
      >
        <Input
          label="Email"
          name="email"
          placeholder="eg: your.name@example.com..."
          type="email"
        />
        <Input
          label="Confirm Email"
          name="confirmEmail"
          placeholder="eg: your.name@example.com..."
          type="email"
        />
        <Input
          label="Password"
          name="password"
          type="password"
        />
        <Input
          label="Confirm Password"
          name="confirmPassword"
          type="password"
        />
        <Input
          label="Name"
          name="name"
        />
        <Input
          label="Surname"
          name="surname"
        />
        <ul>
          <h5>
            Cookies will be used in the following ways:
          </h5>
          <li>
            Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
          </li>
          <li>
            Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
          </li>
        </ul>
        <Fieldset
          classNane="checkboxFieldset"
          legend="Audience Insight Community - Get involved"
        >
          <Checkbox
            checked={true}
            label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
            name="allowContactMe"
            onChange={[Function]}
          />
        </Fieldset>
        <Fieldset
          classNane="checkboxFieldset"
          legend="Terms and conditions"
        >
          <Checkbox
            checked={false}
            label="By signing up, you agree to our terms of service and privacy policy."
            name="tAndC"
            onChange={[Function]}
          />
        </Fieldset>
        <h6>
          The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our 
          <a
            href="#"
          >
            privacy notice
          </a>
        </h6>
        <button
          className="btn btn--cta"
          onClick={[Function]}
        >
          Sign up
        </button>
      </Fieldset>,
      "className": "",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Input
            label="Email"
            name="email"
            placeholder="eg: your.name@example.com..."
            type="email"
          />,
          <Input
            label="Confirm Email"
            name="confirmEmail"
            placeholder="eg: your.name@example.com..."
            type="email"
          />,
          <Input
            label="Password"
            name="password"
            type="password"
          />,
          <Input
            label="Confirm Password"
            name="confirmPassword"
            type="password"
          />,
          <Input
            label="Name"
            name="name"
          />,
          <Input
            label="Surname"
            name="surname"
          />,
          <ul>
            <h5>
              Cookies will be used in the following ways:
            </h5>
            <li>
              Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
            </li>
            <li>
              Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
            </li>
          </ul>,
          <Fieldset
            classNane="checkboxFieldset"
            legend="Audience Insight Community - Get involved"
          >
            <Checkbox
              checked={true}
              label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
              name="allowContactMe"
              onChange={[Function]}
            />
          </Fieldset>,
          <Fieldset
            classNane="checkboxFieldset"
            legend="Terms and conditions"
          >
            <Checkbox
              checked={false}
              label="By signing up, you agree to our terms of service and privacy policy."
              name="tAndC"
              onChange={[Function]}
            />
          </Fieldset>,
          <h6>
            The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our 
            <a
              href="#"
            >
              privacy notice
            </a>
          </h6>,
          <button
            className="btn btn--cta"
            onClick={[Function]}
          >
            Sign up
          </button>,
        ],
        "legend": "Personal Information",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Email",
            "name": "email",
            "placeholder": "eg: your.name@example.com...",
            "type": "email",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Confirm Email",
            "name": "confirmEmail",
            "placeholder": "eg: your.name@example.com...",
            "type": "email",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Password",
            "name": "password",
            "type": "password",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Confirm Password",
            "name": "confirmPassword",
            "type": "password",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Name",
            "name": "name",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Surname",
            "name": "surname",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5>
                Cookies will be used in the following ways:
              </h5>,
              <li>
                Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
              </li>,
              <li>
                Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
              </li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Cookies will be used in the following ways:",
              },
              "ref": null,
              "rendered": "Cookies will be used in the following ways:",
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Analytical purposes – for monitoring usage of the NICE websites in order to improve our services",
              },
              "ref": null,
              "rendered": "Analytical purposes – for monitoring usage of the NICE websites in order to improve our services",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Your preferences – to remember what you view on our websites and to enable us to tailor our services to you",
              },
              "ref": null,
              "rendered": "Your preferences – to remember what you view on our websites and to enable us to tailor our services to you",
              "type": "li",
            },
          ],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Checkbox
              checked={true}
              label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
              name="allowContactMe"
              onChange={[Function]}
            />,
            "classNane": "checkboxFieldset",
            "legend": "Audience Insight Community - Get involved",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "checked": true,
              "label": "The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community",
              "name": "allowContactMe",
              "onChange": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Checkbox
              checked={false}
              label="By signing up, you agree to our terms of service and privacy policy."
              name="tAndC"
              onChange={[Function]}
            />,
            "classNane": "checkboxFieldset",
            "legend": "Terms and conditions",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "checked": false,
              "label": "By signing up, you agree to our terms of service and privacy policy.",
              "name": "tAndC",
              "onChange": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our ",
              <a
                href="#"
              >
                privacy notice
              </a>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "privacy notice",
                "href": "#",
              },
              "ref": null,
              "rendered": "privacy notice",
              "type": "a",
            },
          ],
          "type": "h6",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sign up",
            "className": "btn btn--cta",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Sign up",
          "type": "button",
        },
      ],
      "type": [Function],
    },
    "type": "form",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Fieldset
          legend="Personal Information"
        >
          <Input
            label="Email"
            name="email"
            placeholder="eg: your.name@example.com..."
            type="email"
          />
          <Input
            label="Confirm Email"
            name="confirmEmail"
            placeholder="eg: your.name@example.com..."
            type="email"
          />
          <Input
            label="Password"
            name="password"
            type="password"
          />
          <Input
            label="Confirm Password"
            name="confirmPassword"
            type="password"
          />
          <Input
            label="Name"
            name="name"
          />
          <Input
            label="Surname"
            name="surname"
          />
          <ul>
            <h5>
              Cookies will be used in the following ways:
            </h5>
            <li>
              Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
            </li>
            <li>
              Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
            </li>
          </ul>
          <Fieldset
            classNane="checkboxFieldset"
            legend="Audience Insight Community - Get involved"
          >
            <Checkbox
              checked={true}
              label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
              name="allowContactMe"
              onChange={[Function]}
            />
          </Fieldset>
          <Fieldset
            classNane="checkboxFieldset"
            legend="Terms and conditions"
          >
            <Checkbox
              checked={false}
              label="By signing up, you agree to our terms of service and privacy policy."
              name="tAndC"
              onChange={[Function]}
            />
          </Fieldset>
          <h6>
            The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our 
            <a
              href="#"
            >
              privacy notice
            </a>
          </h6>
          <button
            className="btn btn--cta"
            onClick={[Function]}
          >
            Sign up
          </button>
        </Fieldset>,
        "className": "",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Input
              label="Email"
              name="email"
              placeholder="eg: your.name@example.com..."
              type="email"
            />,
            <Input
              label="Confirm Email"
              name="confirmEmail"
              placeholder="eg: your.name@example.com..."
              type="email"
            />,
            <Input
              label="Password"
              name="password"
              type="password"
            />,
            <Input
              label="Confirm Password"
              name="confirmPassword"
              type="password"
            />,
            <Input
              label="Name"
              name="name"
            />,
            <Input
              label="Surname"
              name="surname"
            />,
            <ul>
              <h5>
                Cookies will be used in the following ways:
              </h5>
              <li>
                Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
              </li>
              <li>
                Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
              </li>
            </ul>,
            <Fieldset
              classNane="checkboxFieldset"
              legend="Audience Insight Community - Get involved"
            >
              <Checkbox
                checked={true}
                label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
                name="allowContactMe"
                onChange={[Function]}
              />
            </Fieldset>,
            <Fieldset
              classNane="checkboxFieldset"
              legend="Terms and conditions"
            >
              <Checkbox
                checked={false}
                label="By signing up, you agree to our terms of service and privacy policy."
                name="tAndC"
                onChange={[Function]}
              />
            </Fieldset>,
            <h6>
              The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our 
              <a
                href="#"
              >
                privacy notice
              </a>
            </h6>,
            <button
              className="btn btn--cta"
              onClick={[Function]}
            >
              Sign up
            </button>,
          ],
          "legend": "Personal Information",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Email",
              "name": "email",
              "placeholder": "eg: your.name@example.com...",
              "type": "email",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Confirm Email",
              "name": "confirmEmail",
              "placeholder": "eg: your.name@example.com...",
              "type": "email",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Password",
              "name": "password",
              "type": "password",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Confirm Password",
              "name": "confirmPassword",
              "type": "password",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Name",
              "name": "name",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Surname",
              "name": "surname",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h5>
                  Cookies will be used in the following ways:
                </h5>,
                <li>
                  Analytical purposes – for monitoring usage of the NICE websites in order to improve our services
                </li>,
                <li>
                  Your preferences – to remember what you view on our websites and to enable us to tailor our services to you
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Cookies will be used in the following ways:",
                },
                "ref": null,
                "rendered": "Cookies will be used in the following ways:",
                "type": "h5",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Analytical purposes – for monitoring usage of the NICE websites in order to improve our services",
                },
                "ref": null,
                "rendered": "Analytical purposes – for monitoring usage of the NICE websites in order to improve our services",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Your preferences – to remember what you view on our websites and to enable us to tailor our services to you",
                },
                "ref": null,
                "rendered": "Your preferences – to remember what you view on our websites and to enable us to tailor our services to you",
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <Checkbox
                checked={true}
                label="The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community"
                name="allowContactMe"
                onChange={[Function]}
              />,
              "classNane": "checkboxFieldset",
              "legend": "Audience Insight Community - Get involved",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "checked": true,
                "label": "The Audience Insight Community helps NICE improve its products and services - please tick this box if you would like to get involved. Find out more about the Audience Insight Community",
                "name": "allowContactMe",
                "onChange": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <Checkbox
                checked={false}
                label="By signing up, you agree to our terms of service and privacy policy."
                name="tAndC"
                onChange={[Function]}
              />,
              "classNane": "checkboxFieldset",
              "legend": "Terms and conditions",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "checked": false,
                "label": "By signing up, you agree to our terms of service and privacy policy.",
                "name": "tAndC",
                "onChange": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our ",
                <a
                  href="#"
                >
                  privacy notice
                </a>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "The information you provide on this form will be used by us to administer your NICE account. For more information about how we process your data, see our ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "privacy notice",
                  "href": "#",
                },
                "ref": null,
                "rendered": "privacy notice",
                "type": "a",
              },
            ],
            "type": "h6",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sign up",
              "className": "btn btn--cta",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Sign up",
            "type": "button",
          },
        ],
        "type": [Function],
      },
      "type": "form",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFourteenAdapter {
      "options": Object {
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
          "getChildContext": Object {
            "calledByRenderer": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
